docs/content/docs/cd/QApps.md

```diff
@@ -9,9 +9,8 @@ type: docs
 
 [QApps Android](http://links.k.avito.ru/qappsAndroid)
 
-Внутреннее хранилище сборок для ручного тестирования.\
-Заливаем в него apk по расписанию из develop.\
-См. конфиг `uploadToQapps` в `build.gradle` приложений.
+This is the internal storage with bulds for manual testing.\
+We publish builds automatically after merge to develop.
```

Comment:
It is a controversial decision. How many builds after develop merge need by users? Maybe the possibility to run TeamCity configuration to upload apk is enough

Reply:
> How many builds after develop merge need by users?

We can't know it in advance. 
Before we supported this contract: "after a successful PR you will find the build in QApps"
I don't see performance or other reasons to break it and make everyone run it manually and wait again.